regjsgen.js

```diff
@@ -97,15 +97,19 @@
   }
 
   // Constructs a string by concatentating the output of each term.
-  function generateSequence(generator, terms) {
+  function generateSequence(generator, terms, separator) {
     var i = -1,
         length = terms.length,
         result = '',
         term;
 
+    if (separator == null) separator = '';
```

Comment:
nit: We could explicitly provide the third parameter and get rid of the condition branch here.

Reply:
Since this function is used in many (6) places and only one needs to specify a separator, I changed the impl to just work with an undefined separator.